src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java

```diff
@@ -437,6 +437,90 @@ private void validateLetterNotifications(List<String> expectedLetterTemplateIds,
     @SuppressWarnings({"Indentation", "unused"})
     private Object[] generateRepsNotificationScenarios() {
         return new Object[]{
+            new Object[]{
+                EVIDENCE_RECEIVED_NOTIFICATION,
+                "paper",
+                Collections.emptyList(),
+                Collections.emptyList(),
+                Arrays.asList("6af62d46-98e5-4ade-aa72-e4a11c56286e", "8eb75404-a442-47aa-bab2-c4ba83a70900"),
+                "no",
+                "no",
+                "no",
+                "no",
+                "0",
+                "0",
+                "2"
+            },
+            new Object[]{
+                EVIDENCE_RECEIVED_NOTIFICATION,
+                "paper",
+                Arrays.asList("7af36950-fc63-45d1-907d-f472fac7af06"),
+                Collections.emptyList(),
+                Arrays.asList("6af62d46-98e5-4ade-aa72-e4a11c56286e"),
+                "no",
+                "no",
+                "yes",
+                "no",
+                "1",
+                "0",
+                "1"
+            },
+            new Object[]{
+                EVIDENCE_RECEIVED_NOTIFICATION,
+                "paper",
+                Arrays.asList("cab48431-a4f0-41f5-b753-2cecf20ab5d4", "7af36950-fc63-45d1-907d-f472fac7af06"),
+                Collections.emptyList(),
+                Collections.emptyList(),
+                "yes",
+                "no",
+                "yes",
+                "no",
+                "2",
+                "0",
+                "0"
+            },
+            new Object[]{
+                EVIDENCE_RECEIVED_NOTIFICATION,
+                "oral",
+                Collections.emptyList(),
+                Collections.emptyList(),
+                Arrays.asList("6af62d46-98e5-4ade-aa72-e4a11c56286e", "8eb75404-a442-47aa-bab2-c4ba83a70900"),
+                "no",
+                "no",
+                "no",
+                "no",
+                "0",
+                "0",
+                "2"
+            },
+            new Object[]{
+                EVIDENCE_RECEIVED_NOTIFICATION,
+                "oral",
+                Arrays.asList("30260c0b-5575-4f4e-bce4-73cf3f245c2d"),
+                Collections.emptyList(),
+                Arrays.asList("6af62d46-98e5-4ade-aa72-e4a11c56286e"),
+                "no",
+                "no",
+                "yes",
+                "no",
+                "1",
+                "0",
+                "1"
+            },
+            new Object[]{
+                EVIDENCE_RECEIVED_NOTIFICATION,
+                "oral",
+                Arrays.asList("c5654134-2e13-4541-ac73-334a5b5cdbb6", "30260c0b-5575-4f4e-bce4-73cf3f245c2d"),
+                Collections.emptyList(),
+                Collections.emptyList(),
+                "yes",
+                "no",
+                "yes",
+                "no",
+                "2",
+                "0",
+                "0"
```

Comment:
According to the ticket, regardless of the subscription for the appellant/rep, this letter should be sent to them.